docs/feature_converters.md

```diff
@@ -1,12 +1,10 @@
 # Converters
 
-Since many drop-in replacement controllers now exist, we've done our best to make them easy to use in existing designs.
+This page documents the automated process for converting keyboards to use drop-in replacement Pro Micro controllers. This process is designed to be easy to use and can be completed in a few simple steps.
```

Comment:
The process is not just for "Pro Micro", for instance I have a drop in replacement for blackpill (be it work-in-progress and may never see public use).

```suggestion
This page documents the automated process for converting keyboards to use drop-in replacement controllers. This process is designed to be easy to use and can be completed in a few simple steps.
```